nixos/modules/services/misc/snapper.nix

```diff
@@ -142,7 +150,7 @@ in
     systemd.timers.snapper-timeline = {
```

Comment:
I think we can remove `systemd.timers.snapper-timeline` and instead add `startAt` to `systemd.services.snapper-timeline`.